build/media_source/system/js/fields/pagebuilder/components/app.vue

```diff
@@ -73,7 +73,7 @@
         const deviceOrder = Object.keys(this.resolution);
         const activeIndex = deviceOrder.indexOf(this.activeDevice);
         const styles = {
-          'min-width': activeIndex === 0 ? 0 : this.resolution[this.activeDevice],
+          'min-width':'300px;', //activeIndex === 0 ? 0 : this.resolution[this.activeDevice],
```

Comment:
This not a coding style. Either you can remove or add code don't leave after comment out.